webhookentitytype.go

```diff
@@ -0,0 +1,12 @@
+package monta
+
+// WebhookEtityType represents the type of a webhook entity.
+type WebhookEntityType string
+
+// Known [WebhookEtityType] values.
+const (
```

Comment:
They generally support more than this right? But these are the only ones that are relevant right now?

Reply:
There is and they will add more from time to time. These were the ones available when this was implemented.